### PR TITLE
feat: add partners and general merch

### PR DESCRIPTION
**Pull request by aleksKudr**
**🟢 Add `deploy` label if you want to deploy this Pull Request to staging environment**

#### 🧑‍⚖️ Pull Request Naming Convention

- Title should follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- Do not put issue id in title
- Do not put WIP in title. Use [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) functionality
- Consider to add `area:*` label(s)

* [ ] I followed naming convention rules

---

#### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Test Case
- [ ] Documentation update
- [ ] Other

#### 🔗 Related issue link

[Partners](https://github.com/rolling-scopes/rs-site/issues/13)
[Merch](https://github.com/rolling-scopes/rs-site/issues/22)

#### 💡 Background and solution

_Describe the big picture of your changes here_

#### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Database migration is added or not needed
- [ ] Documentation is updated/provided or not needed
- [ ] Changes are tested locally

### CODE REVIEW

File: types/index.ts [hunk 1]
```diff
@@ -25,6 +25,17 @@ export type SpeakersT = Base & Link;
 
 export type DonationV1T = Base & Link;
 
+export type MerchGeneralT = Base & Link;
+
+export type PartnersT = {
+  title: string;
+  companies: {
```
Comment (oleksiisydorenko1) [inline types/index.ts:32]: We need to add 1 extra field, **priority** or something like that to determine most valuable partners (for example EPAM should be with priority 1 and should go first in the list)

**Comment by oleksiisydorenko1** [comment]
@aleksKudr please don't forget to deploy the studio after mergin, because currently, changes are not visible on https://locaclhost.sanity.studio/studio/desk

**Review by oleksiisydorenko1 — APPROVED** [review]

File: components/MerchGeneral/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
+.block {
+  @apply flex flex-col justify-center h-80 text-center px-8;
+  // background-color: #191b1d;
```
Comment (crzoukman) [inline components/MerchGeneral/styles.module.scss]: a comment?

File: components/MerchGeneral/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  @apply text-slate-900 text-4xl tracking-tight font-extrabold sm:text-5xl;
+}
+
+.description {
+  @apply mt-6 text-base w-3/5;
+}
+
+.link {
+  @apply inline-flex justify-center text-sm font-semibold py-2.5 px-4 text-white mt-6 ml-8;
+  background-color: #191b1d;
```
Comment (crzoukman) [inline components/MerchGeneral/styles.module.scss]: consider using tailwind ` bg-[#191b1d]`

File: components/Partners/styles.module.scss [hunk 4]
```diff
@@ -0,0 +1,22 @@
+.block {
+  @apply flex flex-col justify-center h-96 text-center px-8 dark:text-white;
+  background-color: #191b1d;
```
Comment (crzoukman) [inline components/Partners/styles.module.scss]: consider using tailwind `bg-[#191b1d]`

File: components/Partners/styles.module.scss [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+.companies {
+  @apply flex items-center;
+}
+
+.link {
+  @apply inline-flex justify-center text-sm font-semibold py-2.5 px-4 ml-8;
+  color: #191b1d;
+  position: relative;
```
Comment (crzoukman) [inline components/Partners/styles.module.scss]: consider using tailwind `relative text-[#191b1d]`

**Review by crzoukman — APPROVED** [review]

**Pull request merged**